docs/source/EccFramework.rst

```diff
@@ -0,0 +1,154 @@
+ECC Framework
+======================================================================================================
```

Comment:
This headline is too long. Sphinx should tell you that in a warning. Please inspect the build logs of RtD (by clicking on the CI run)

Reply:
I reduced the number of "=" signs.

Reply:
See the other comment and the build log. The number of `=` has to match the title length.

Reply:
Done